hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMSecurityProtocolServer.java

```diff
@@ -421,9 +421,9 @@ public void stop() {
   }
 
   public void join() throws InterruptedException {
-    LOGGER.trace("Join RPC server for SCMSecurityProtocolServer.");
+    LOGGER.info("Join RPC server for SCMSecurityProtocolServer.");
```

Comment:
I am a bit unsure about these two... do we really want/need to increase the log level for these messages?

Reply:
This join() is called only once, right after stop().  The log level in stop is info. Why it's trace in join.  I think we should use the same level as stop.